src/test/java/converter/StringToUserConverterTest.java

```diff
@@ -0,0 +1,71 @@
+package converter;
+
+import model.User;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class StringToUserConverterTest {
+    private final StringToUserConverter stringToUserConverter = new StringToUserConverter();
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testConvertEmptyString() {
+        stringToUserConverter.convert("");
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testConvertBlankString() {
+        String actual = " ";
```

Comment:
testComvertEmptyString doesn’t have actual
But this methods are the same. Should looks the same